packages/angular/projects/clr-angular/src/data/datagrid/datagrid-column-separator.ts

```diff
@@ -4,74 +4,160 @@
  * The full license information can be found in LICENSE in the root directory of this project.
  */
 import { DOCUMENT } from '@angular/common';
-import { Component, Inject, Renderer2 } from '@angular/core';
-
-import { ClrDragEvent } from '../../utils/drag-and-drop/drag-event';
+import { AfterViewInit, Component, ElementRef, Inject, NgZone, OnDestroy, Renderer2, ViewChild } from '@angular/core';
+import { KeyCodes, IEKeyCodes } from '../../utils/enums/key-codes.enum';
+import { ClrCommonStringsService } from '../../utils/i18n/common-strings.service';
+import { UNIQUE_ID, UNIQUE_ID_PROVIDER } from '../../utils/id-generator/id-generator.service';
 import { ColumnResizerService } from './providers/column-resizer.service';
 import { TableSizeService } from './providers/table-size.service';
-import { UNIQUE_ID_PROVIDER, UNIQUE_ID } from '../../utils/id-generator/id-generator.service';
+
+// Default resize length on each keyboard move event
+const KEYBOARD_RESIZE_LENGTH = 12;
```

Comment:
12?

Reply:
This is how much pixel movement would happen per keydown event, basically length/keydown_event. I have chosen this number based on:
- Our baseline is 24px. So something that can divide our baseline evenly.
- UX-wise it seems to give a good resize speed on keydown (Might be subjective though)